src/WeatherCalendar.h

```diff
@@ -0,0 +1,20 @@
+#include "Weather.h"
+
+class WeatherCalendar {
+public:
+    WeatherCalendar();
+    WeatherCalendar(int len);
+    ~WeatherCalendar();
+    void addDayWeather(Weather weather,unsigned int Index);
+    Weather getWeatherIndex(unsigned int Index);
+
+    void findMaxTempratureByDay(weekDay day);
+    void isLvivWeather(double humidity, type Wtype);
+    void SelectionSortByDays();
+    void SelectionSortByTemperature();
```

Comment:
All fields and methods should start from lower case

Reply:
changed